### PR TITLE
PR for ETL article for Review

### PR DESCRIPTION
**Pull request by adamboeglin**

### CODE REVIEW

File: docs/example-scenario/data/hybrid-etl-with-adf.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: Hybrid ETL with existing on-premises SSIS and Azure Data Factory v2
+description: Hybrid ETL with existing on-premises SSIS and Azure Data Factory v2
+author: alhieng
+ms.date: 9/20/2018
+---
+
+# Hybrid ETL with existing on-premises SSIS and Azure Data Factory v2
```
Comment (adamboeglin) [inline docs/example-scenario/data/hybrid-etl-with-adf.md]: Is the "V2" necessary in the name?

File: docs/example-scenario/data/hybrid-etl-with-adf.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+The decision is much easier when you already have existing SSIS packages that have on-premise dependencies such as data sources, or files which are not accessible from Azure.  In this scenario, your only option is Self-hosted IR. This approach provides the most flexibility to leverage the cloud as the orchestration engine, without having to rewrite existing packages.
+
+Ultimately, the intent is to move the processed data into the cloud for further refinement or to combine with other data stored in the cloud.  As part of the design process, keep track of the number of activities used in the ADF pipelines.  You can learn more about [pipelines and activities](https://docs.microsoft.com/en-us/azure/data-factory/concepts-pipelines-activities).
+
+### Availability, Scalability, and Security
```
Comment (adamboeglin) [inline docs/example-scenario/data/hybrid-etl-with-adf.md]: Need more detail here

File: docs/example-scenario/data/hybrid-etl-with-adf.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+## Pricing
+
+Azure Data Factory is a very cost-effective way to orchestrate data movement in the cloud.  The cost is based on the several factors.
+
+* Number of pipelines executions
+* Number of entities/activities used within the pipeline
+* Number of monitoring operations
+* Number of Integration Runs (Azure-hosted IR or Self-hosted IR)
+
+ADF uses a consumption-based billing; therefore, cost is only incurred during pipeline executions and monitoring.  The execution of a basic pipeline would cost as little as $0.50 cents and the monitoring as little as $25 cents. The [Azure cost calculator](https://azure.microsoft.com/en-us/pricing/calculator/) can be used to create a more accurate estimate based on your specific work load.
```
Comment (adamboeglin) [inline docs/example-scenario/data/hybrid-etl-with-adf.md]: Let's link to the calculator with the resources already added to it.

File: docs/example-scenario/data/hybrid-etl-with-adf.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+When running a hybrid ETL workload, you must factor in the cost of the Virtual machine used to host your SSIS packages. This is explained in the above considerations section of this document. This cost is based on the size of the VM ranging from a D1v2 (1 core, 3.5 GB RAM, 50GB Disk) to E64V3 (64 cores, 432GB RAM, 1600GB Disk).  If you need further guidance on selection the appropriate VM size refer to [VM performance considerations](https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-sizes-specs#performance-considerations).
+
+## Next Steps
+
+* To learn more about [Azure Data Factory](https://azure.microsoft.com/en-us/services/data-factory/)
+* Get started with Azure Data Factory by following the [Step-by-Step tutorial](https://docs.microsoft.com/en-us/azure/data-factory/#step-by-step-tutorials)
+* [Provision the Azure-SSIS Integration Runtime in Azure Data Factory](https://docs.microsoft.com/en-us/azure/data-factory/tutorial-deploy-ssis-packages-azure)
```
Comment (MikeWasson) [inline docs/example-scenario/data/hybrid-etl-with-adf.md]: Consider add a link to our [ADF reference architecture](https://docs.microsoft.com/en-us/azure/architecture/reference-architectures/data/enterprise-bi-adf) for an example that uses Analysis Service in Azure

File: docs/example-scenario/data/hybrid-etl-with-adf.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+* [SQL Data Warehouse][docs-sql-data-warehouse] is used to centralize data in the cloud for easy access using standard ANSI SQL queries.
+
+### Alternatives
+
+Data Factory could invoke data cleansing procedures implemented using other technologies, such as a Databricks notebook, Python script, or SSIS instance running in a virtual machine. [Install paid or licensed custom components on Azure-hosted IR][] may be an alternative to the hybrid approach.
+
+## Considerations
+
+Integrated Runtime (IR) supports two models, self-hosted IR or Azure-hosted IR.  You must first decide between these two options.  Self-hosting is more cost effective but requires more overhead for you to maintain and manage. You can learn more at [Self-hosted IR](https://docs.microsoft.com/en-us/azure/data-factory/concepts-integration-runtime#self-hosted-integration-runtime).  If you need help determining which IR to use you can read more on “[Determine which IR to use](https://docs.microsoft.com/en-us/azure/data-factory/concepts-integration-runtime#determining-which-ir-to-use)”
```
Comment (MikeWasson) [inline docs/example-scenario/data/hybrid-etl-with-adf.md]: For links to docs.microsoft.com, just use the URL path (not including the locale)

e.g., `/azure/data-factory/concepts-integration-runtime#determining-which-ir-to-use`

File: docs/example-scenario/data/hybrid-etl-with-adf.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+* Number of monitoring operations
+* Number of Integration Runs (Azure-hosted IR or Self-hosted IR)
+
+ADF uses a consumption-based billing; therefore, cost is only incurred during pipeline executions and monitoring.  The execution of a basic pipeline would cost as little as $0.50 cents and the monitoring as little as $25 cents. The [Azure cost calculator](https://azure.microsoft.com/en-us/pricing/calculator/) can be used to create a more accurate estimate based on your specific work load.
+
+When running a hybrid ETL workload, you must factor in the cost of the Virtual machine used to host your SSIS packages. This is explained in the above considerations section of this document. This cost is based on the size of the VM ranging from a D1v2 (1 core, 3.5 GB RAM, 50GB Disk) to E64V3 (64 cores, 432GB RAM, 1600GB Disk).  If you need further guidance on selection the appropriate VM size refer to [VM performance considerations](https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-sizes-specs#performance-considerations).
+
+## Next Steps
+
+* To learn more about [Azure Data Factory](https://azure.microsoft.com/en-us/services/data-factory/)
```
Comment (MikeWasson) [inline docs/example-scenario/data/hybrid-etl-with-adf.md]: remove locale string (en-us) from URL

**Pull request merged**